certified-mail-sender-models/src/main/scala/it/pagopa/interop/certifiedMailSender/model/InteropEnvelope.scala

```diff
@@ -0,0 +1,13 @@
+package it.pagopa.interop.certifiedMailSender.model
+
+import java.util.UUID
+
+final case class InteropEnvelope(
+  id: UUID,
+  to: List[String],
+  cc: List[String],
+  bcc: List[String],
+  subject: String,
+  body: String,
+  attachments: List[MailAttachment]
```

Comment:
It wouldn't be bad to define `MailAttachment` in this file too, and avoid at all the creation of 2 separate files